lang/test/org/partiql/lang/eval/EvaluatingCompilerCastTest.kt

```diff
@@ -921,7 +960,11 @@ class EvaluatingCompilerCastTest : EvaluatorTestBase() {
                 assertEquals(castCase.expectedErrorCode, p.errorCode)
             }
         }
-        else -> assertEquals(castCase.expected, eval(castCase.expression).toString())
+        else -> assertEquals(castCase.expected,
+                             eval(source = castCase.expression,
+                                  session = castCase.session
+                             ).toString()
+        )
```

Comment:
nit: I find this formatting slightly confusing. Consider:

```suggestion
        else -> assertEquals(
            castCase.expected,
            eval(source = castCase.expression, session = castCase.session).toString()
        )
```